sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/persons/PersonDirectorySteps.java

```diff
@@ -64,38 +71,136 @@ public PersonDirectorySteps(
+              CASE_GRID_RESULTS_ROWS, apiState.getLastCreatedPerson().getUuid());
+          assertHelpers.assertWithPoll20Second(
+              () ->
+                  Truth.assertWithMessage(
+                          apiState.getLastCreatedPerson().getUuid()
+                              + " value is not displayed in grid Disease column")
+                      .that(apiState.getLastCreatedPerson().getUuid())
+                      .isEqualTo(
+                          String.valueOf(
+                              webDriverHelpers.getTextFromPresentWebElement(
+                                  CASE_PERSON_ID_COLUMN_HEADERS))));
+        });
+
+    Then(
+        "I check that number of displayed Person results is {int}",
+        (Integer number) ->
+            assertHelpers.assertWithPoll20Second(
+                () ->
+                    Assert.assertEquals(
+                        webDriverHelpers.getNumberOfElements(CASE_GRID_RESULTS_ROWS),
+                        number.intValue(),
+                        "Number of displayed persons is not correct")));
+
+    Then(
+        "I check that number of displayed Persons results is {int}",
+        (Integer number) ->
+            assertHelpers.assertWithPoll20Second(
+                () ->
+                    Assert.assertEquals(
+                        webDriverHelpers.getNumberOfElements(CASE_GRID_RESULTS_ROWS),
+                        number.intValue(),
+                        "Number of displayed cases is not correct")));
+
+    Then(
+        "I change Year of birth filter to {string}",
+        (String yearOfBirth) ->
+            webDriverHelpers.selectFromCombobox(BIRTH_YEAR_COMBOBOX, yearOfBirth));
+
+    Then(
+        "I change Month of birth filter to {string}",
+        (String monthOfBirth) ->
+            webDriverHelpers.selectFromCombobox(BIRTH_MONTH_COMBOBOX, monthOfBirth));
+
+    Then(
+        "I change Day of birth filter to {string}",
+        (String dayOfBirth) -> webDriverHelpers.selectFromCombobox(BIRTH_DAY_COMBOBOX, dayOfBirth));
+
+    Then(
+        "I change present condition filter to {string}",
+        (String presentCondition) ->
+            webDriverHelpers.selectFromCombobox(PRESENT_CONDITION, presentCondition));
+
+    Then(
+        "I change REGION filter to {string}",
+        (String region) -> webDriverHelpers.selectFromCombobox(REGIONS_COMBOBOX, region));
+
+    Then(
+        "I change DISTRICT filter to {string}",
+        (String district) -> webDriverHelpers.selectFromCombobox(DISTRICTS_COMBOBOX, district));
+
+    Then(
+        "I change Community filter to {string}",
+        (String community) ->
+            webDriverHelpers.selectFromCombobox(COMMUNITY_PERSON_COMBOBOX, community));
+
     Then(
         "I search after last created person from API by {string}",
         (String searchCriteria) -> {
           String searchText = "";
-          String personUUID = apiState.getLastCreatedPerson().getUuid();
```

Comment:
we should not change existing methods because this will cause failures into all other tests using it